### DISCUSSION TITLE
Using cross-env, Fail in next

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#34637] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

无

### What browser are you using? (if relevant)

chrome

### How are you deploying your application? (if relevant)

next start

### Describe the Bug

There are four environments in our project. At present, the environment of next does not meet our needs. We need to switch between different environments

### Expected Behavior

There are four environments in our project. At present, the environment of next does not meet our needs. We need to switch between different environments

### To Reproduce

There are four environments in our project. At present, the environment of next does not meet our needs. We need to switch between different environments

**Comment by anhthien-bic** [comment]
I also face the issue

**Comment by parlay96** [comment]
@thienna 
![image](https://user-images.githubusercontent.com/32978177/154911599-3879908d-5f05-40b1-b23e-e37f05b6c4ec.png)
You can do it. You can try this
![image](https://user-images.githubusercontent.com/32978177/154911721-817b9d15-20dd-4309-a332-e5cee528395a.png)
